data/helpers/conversations/upsertConversations.ts

```diff
@@ -77,10 +81,20 @@ const setupAndSaveConversations = async (
   });
 
   const conversationsToUpsert: Conversation[] = [];
-  conversations.forEach((conversation) => {
+  conversations.forEach(async (conversation) => {
     const alreadyConversationInDbWithTopic =
       alreadyConversationsByTopic[conversation.topic];
 
+    // If spam score is not computed, compute it
+    if (
+      conversation.spamScore === undefined ||
+      conversation.spamScore === null
```

Comment:
are we positive every conversation will get a spam score? We used to not score empty convos but I see that you removed it. Just want to make sure that this won't execute in a loop for some convos. What score will an empty conversation get?

Reply:
@nmalzieu These diffs on spamScore.ts should make sure every conversation gets scored, right?

https://github.com/ephemeraHQ/converse-app/pull/788/files/39d6fc1b2b170953eb0242b427d57fd86865e3d4#diff-22463bf42246ca396975d8b9e611855052c26a367b7cfc39609b9ee304cfb889L51

https://github.com/ephemeraHQ/converse-app/pull/788/files/39d6fc1b2b170953eb0242b427d57fd86865e3d4#diff-22463bf42246ca396975d8b9e611855052c26a367b7cfc39609b9ee304cfb889R82

Reply:
Looks like every conv should get a spam score. Added a debug log here so if we do get in a loop, it will be easy to diagnose. Initial tests looks good!